src/components/dashboard.tsx

```diff
@@ -0,0 +1,416 @@
+import { Fade, makeStyles } from '@material-ui/core';
```

Comment:
I had to use some more obscure git command (`merge-file`), the default merge though that `app.tsx` is modified instead of renamed, changing the rename threshold doesn't help. I hope this won't cause too much trouble with other PRs when this is merged.